packages/web/components/Badge/Badge.tsx

```diff
@@ -38,26 +181,23 @@ const getBadgeDescription = (badgeType: BadgeType): string =>
   `${getBadgeCopySubpath(badgeType)}.description`
 
 const BadgeComponent: React.FC<Props> = ({ badge }) => {
-  const { t } = useTranslation('common')
+  const { t } = useTranslation('badge')
```

Comment:
You'll want to add 'badge' to the `namespacesRequired` array on the page component so these translations get included in the initial page load

Reply:
Great catch, thank ya!